evaluations/data/evaluation_configs.py

```diff
@@ -77,10 +77,131 @@ def _smoke_test(num_runs=100):
           )
       )
 
+
+def _test_with_selected_parameters_second_part(num_runs=100):
+  """Configurations with handy selected parameters for scenarios 3b - 5.
+
+  Args:
+    num_runs: the number of runs per scenario * parameter setting.
+
+  Returns:
+    An EvaluationConfig.
+  """
+  scenario_config_list = []
+  # Common parameters
+  universe_size = 100000
```

Comment:
Best practice is to either define this is as a constant or pass it as a parameter, possibly with a default value.

Reply:
Thanks Matthew. Per discussion with Xichen, will define these as parameters, and put default values as global variables.

Reply:
Done.